news/templates/news/admin_events.html

```diff
@@ -15,50 +15,49 @@ <h1 class="ui text-centered header">
                 <i class="ui green add icon"></i>
             </a>
         </h1>
-        <div class="ui four  doubling stackable cards">
+        <div class="ui four doubling stackable cards">
             {% for event in events %}
                 <div class="ui card">
-                    <div class="image">
+                    <a class="image" href="{% url "event" event.id %}">
                         <img src="{{ event.image.url }}">
-                    </div>
+                    </a>
                     <div class="content">
-                        <div class="header">{{ event.title }}</div>
+                        <a class="header" href="{% url "event" event.id %}">{{ event.title }}</a>
                         <div class="meta">
-                            {% if event.get_future_occurrences %}
-                                {{ event.get_future_occurrences.count }} {% trans "Future occurrences" %}
-                            {% endif %}
-                        </div>
-                        <div class="description">{{ event.clickbait }}</div>
-                    </div>
-                    {% if event.featured or event.hidden or event.private %}
-                        <div class="extra content">
+                            {% with event.get_future_occurrences.count as num_future_occurrences %}
```

Comment:
This will only count published `TimePlace`s. I was confused for a bit because it showed no future occurences even though the event had one. The event was hidden so `get_future_occurrences()` returned nothing.

Maybe count all occurrences?

Reply:
I hadn't really considered that, as I just reused the code from earlier, but that is better. I have changed it now.